javascript/javascript1/week3/myFavoriteSongs.js

```diff
@@ -0,0 +1,62 @@
+const songDatabase = [{
+    songId: 1,
+    title: 'My baby',
+    artist: 'Soggy socks',
+  },
+  {
+    songId: 2,
+    title: '3 nails in wood',
+    artist: 'The carpenters',
+  },
+  {
+    songId: 3,
+    title: 'Blacker than black',
+    artist: 'Instant coffee',
+  },
+  {
+    songId: 4,
+    title: 'When is enough too little?',
+    artist: 'The spies girls',
+  },
+];
+const myPlaylist = [];
+
+function myList(index)
+    {
+        const newSong=songDatabase[index];
+        myPlaylist.push(newSong);
+    }
+console.log("My List:");
+
+function addSongToDatabase(song) 
+{
+    songDatabase.push(song);
+}
+
+  function getSongByTitle(title)
+   {
+    index = songDatabase.findIndex(x => x.title === title);
```

Comment:
and what if you don't find it?

Reply:
I changed the code here..
Please check here...https://github.com/Aswini-D3/hyf-homework/blob/javascript/javascript1/week3/javascript/javascript1/week3/myFavoriteSongs.js